### PR TITLE
Celery set prefetch count on restart

### PR DESCRIPTION
**Pull request by dobosevych**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

### CODE REVIEW

File: celery/worker/consumer/consumer.py [hunk 1]
```diff
@@ -649,7 +649,7 @@ def on_task_received(message):
 
         return on_task_received
 
-    def _restore_prefetch_count_after_connection_restart(self, p):
+    def _restore_prefetch_count_after_connection_restart(self, p, *args):
```
Comment (dobosevych) [inline celery/worker/consumer/consumer.py:652]: @thedrow based on my test, I need to add *args as an argument
```
>>> import vine
>>> from vine.promises import promise
def raise_exception():
    raise KeyError('Foo')

>>> p = promise(raise_exception, on_error=print)
>>> p()
Foo
```

File: celery/worker/consumer/consumer.py [hunk 2]
```diff
@@ -622,9 +622,9 @@ def on_task_received(message):
                 try:
                     ack_log_error_promise = promise(call_soon, (message.ack_log_error,),
                                                     on_error=self._restore_prefetch_count_after_connection_restart)
-                    reject_log_error_promise = promise(call_soon,
-                                                       (message.reject_log_error,),
-                                                       on_error=self._restore_prefetch_count_after_connection_restart)
+                    reject_log_error_promise = \
```
Comment (thedrow) [inline celery/worker/consumer/consumer.py:625]: Is there something we can do with this code formatting? It's not readable.

**Review by thedrow — CHANGES_REQUESTED** [review]

File: celery/worker/consumer/consumer.py (hunk 2 above)
Comment (dobosevych) [reply celery/worker/consumer/consumer.py:625]: agree, I will improve that
whether there are plans to use something like black?

Comment (dobosevych) [reply celery/worker/consumer/consumer.py]: improved formating

**Review by auvipy — APPROVED** [review]

**Pull request merged**